ai_genomics/getters/crunchbase.py

```diff
@@ -59,3 +59,13 @@ def get_crunchbase_ai_genomics_entity_groups(k: int = 500) -> pd.DataFrame:
     """
     fname = f"inputs/entities/crunchbase_entity_group_vectors_k_{k}.csv"
     return load_s3_data(bucket_name, fname)
+
+
+def get_crunchbase_ai_genomics_description_embeddings() -> pd.DataFrame:
+    """Gets an array of description embeddings and the associated company IDs.
+
+    Returns:
+       pd.DataFrame: Description embeddings and the associated company IDs.
+    """
+    fname = "inputs/embedding/pat_embeddings.csv"
```

Comment:
this is getting the patent embeddings

Reply:
all these getters assume that the descriptions are about ai_genomics (not ai only, or not genomics only) - given that your script doesn't specify this nor do the embedding file names, I would change this to` get_DATASOURCE_description_embeddings` - unless you've only embedded ai genomics documents? then I would make the appropriate name changes to oa.json, gtr.json etc. so that its clearer.

Reply:
Yeah, I've only embedded ai and genomics docs, so changing the file name is a good idea.